connectors/servicenow/src/test/java/com/vmware/connectors/servicenow/ServiceNowControllerTests.java

```diff
@@ -54,7 +54,6 @@ public void setup() throws Exception {
     @Test
     public void testProtectedResource() throws Exception {
         testProtectedResource(POST, "/cards/requests");
-        testProtectedResource(GET, "/success?code=test-code");
```

Comment:
This was from when I didn't know the details of how our client code handles OAuth.  Since the request is rejected with a 401 Unauthorized before we even route to the controller, this non-existent path still passes and went by unnoticed.